### ISSUE TITLE
[🐛 Bug]: Regression (worked in 3.14) - javascript error: this.each is not a function - Failing to parse elements in text editors

### ISSUE THREAD

**Issue by jmalmsten-panopto** [SeleniumHQ/selenium#10326]
### What happened?

Hi,

I upgraded from Selenium 3.14 to 4 and almost everything is going smoothly except when i attempt to check in any elements exist or are clickable inside a mce toolbar(wysiwyg editor) I get the following error: 

OpenQA.Selenium.JavaScriptException : javascript error: this.each is not a function
      (Session info: chrome=97.0.4692.99)

These tests passed before upgrading. 

I noticed a similar error in Chromedriver 81 and was wondering if anyone knew any workarounds for this or know if this is a known issue?

I filed a bug with Chromedriver which I linked below, but they said this is a Selenium issue.

Thanks,
Joe Malmsten

https://bugs.chromium.org/p/chromedriver/issues/detail?id=4017#c1

### How can we reproduce the issue?

```shell
1. Have a webpage with a MCE toolbar (the one I'm using is from Blackboard classic)
2. Attempt to check if any element in the toolbar is displayed or clickable. 

Expected: The WebElements are retrieved properly.
Actual: We get the this.each javascript error.
```

### Relevant log output

```shell
Message: 
    OpenQA.Selenium.JavaScriptException : javascript error: this.each is not a function
      (Session info: chrome=98.0.4758.82)

  Stack Trace: 
    WebDriver.UnpackAndThrowOnError(Response errorResponse)
    WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
    WebDriver.InternalExecute(String driverCommandToExecute, Dictionary`2 parameters)
    WebElement.Execute(String commandToExecute, Dictionary`2 parameters)
    WebElement.get_Displayed()
    EventFiringWebElement.get_Displayed()
    WebDriverExtension.ElementIsDisplayed(IWebDriver driver, Func`2 elementFunction) line 55
    CourseForumPage.OpenSessionPicker(Boolean usingSaasEditor, CoursePage coursePage) line 195
    BlackboardTests.BBVerifyEmbed(InstanceName instance) line 457
```

### Operating System

Windows 10

### Selenium version

C# 4.1.0

### What are the browser(s) and version(s) where you see this issue?

Chrome 97

### What are the browser driver(s) and version(s) where you see this issue?

chromedriver 97.0.4692.99

**Comment by titusfortner** [comment]
This was a bug that is specific to Chromedriver 81 and was based on a compilation issue of the a JS atom used in the driver. https://bugs.chromium.org/p/chromedriver/issues/detail?id=3422

If you are using Selenium 4.1, then you should be working with a w3c compliant session, which means executing the `Displayed` getter method will execute the atom generated by Selenium from the client side. Does this error happen every time you use that method, or only with the MCE toolbar? Does it work in Firefox?

**Comment by jmalmsten-panopto** [comment]
Hi Titus, 

I saw the Chromedriver 81 issue which was why I opened the intial but at the Chromedriver site and they said it was a Selenium issue. Looks like this bug came back by Chromedriver 97 which is what I am testing on.

After downloading geckodriver you are correct that the geckodriver can still interact with the text editor. I will bring this update to the Chromedriver teams' attention. 

Thanks,
Joe
